lib/dynamic_screening_solutions/api.py

```diff
@@ -333,6 +333,8 @@ def get_forms_by_division(self, division_id):
         request_url = self.get_request_url(resource_path=resource_path)
         response = self.request_get(request_url)
         forms = response.json()
+        if 'error' in forms and forms['error'] is True:
+            forms = []
```

Comment:
```suggestion
        try:
            forms = response.json()
            if type(forms) == 'dict' and forms.get('error'):
                forms = []
      except Exception:
          forms = None
```

Reply:
Done

Reply:
@jontsai the try/except is not added for many functions in the file `lib/dynamic_screening_solutions/api.py`.
Should I add a similar try/except handle for other functions as well?